_pages/install/smartEdit-setup-instructions-for-spartacus.md

```diff
@@ -159,3 +170,39 @@ The following steps are for configuring SmartEdit to work without the SmartEdit
    By starting your app in SSL mode, you avoid an `unsafe scripting` message from the browser.
 
    **Note:** If you start your application without using SSL mode, the two references to `https://localhost:4200` must be changed to `http://localhost:4200`.
+
+## Troubleshooting
+
+When you are in SmartEdit, if you have not opened any actual Spartacus pages (such as the Homepage), then any issues you might encounter are not caused by Spartacus. For example, on the **Your Site** page, Spartacus is not actually opened on this page. If you see an issues (such as the site list not containing any base sites), please contact the SmartEdit team for help.
+
+### The Perspective Toolbar is Missing
+
+The perspective toolbar will be missing if `webApplicationInjector.js` is not loaded. Please see the procedures above for information on how to copy and set `webApplicationInjector.js`.
+
+The following example shows the perspective toolbar not appearing when `webApplicationInjector.js` is not loaded.
+
+![webApplicationInjector.js]({{ site.baseurl }}/assets/images/webApplicationInjector.png)
+
+### The Preview URL Does Not Match Your Website URL
+
+SmartEdit opens the `previewUrl` in its iframe. You can open the `previewUrl` in a browser to see whether it works. Although it is recommended to start the app in SSL mode, you are not required to do so. You only need to make sure that the preview URL matches your site URL.
+
+### The Page is Not Refreshed When You Add, Edit, or Remove Components
+
+When you add, edit, or remove components, the CMS page data is reloaded. If the page does not refresh, it may be caused by the caching in OCC.
+
+You can verify this by opening the **Network** tab in your browser (right-click > **Inspect**), and then editing a component. When you do this, you should see a new request to load CMS page data. Check the response of this request to see whether the edited component data is updated.
+
+If only some operations do not make the page refresh (for example, editing a component causes the page to refresh, but updating an image does not cause the page to refresh), then please contact the SmartEdit team for help.
```

Comment:
This sentence `then please contact the SmartEdit team for help.`, can be this: `please download the latest webApplicationInjector.js from the latest patch release of backend`.

Reply:
👍